docs/user/whats-new.asciidoc

```diff
@@ -1,12 +1,255 @@
+
+[float]
+==== Saved query and Add filter menus
+
+Querying and filtering your Elastic data is now easier with the improved *Saved query*
+and *Add filter* menus. The *Saved query* menu consolidates all of your
+query and filter needs, including the option to change your filtering language.
+
+[role="screenshot"]
+image::images/highlights-saved-query-menu.png[Menu for saving queries and adding filters, width=50%]
+
+[float]
+[[highlights-8.3-console]]
+=== Support for {kib} APIs in Console
+
+**Dev Tools > Console** now supports sending requests to {kib} APIs.
+Prepend any {kib} API endpoint with `kbn:`
+and send the request.
+
+[role="screenshot"]
+image::images/highlights-console.png[Console showing how to send requests to Kibana APIs]
+
+
+[float]
+[[highlights-8.3-vis-editors]]
+=== Enhancements to visualization editors
+
+[float]
+==== Range annotations in Lens
+
+Unlock insights by visualizing event data directly on your *Lens* visualizations.
+With range annotations, you can show windows of time such as releases,
+maintenance windows, holidays, and more to provide context to metric performance.
+For information about *Lens* annotations, check {kibana-ref}/lens.html#add-annotations[Add annotations].
+
+[role="screenshot"]
+image::images/highlights-lens-range-inside.png[Range annotation inside of the visualization]
+
+[.text-center]
+*Range annotation inside a visualization*
+
+[role="screenshot"]
+image::images/highlights-lens-range-outside.png[Range annotation outside of the visualization]
+
+[.text-center]
+*Range annotation outside of a visualization*
+
+[float]
+==== Anomaly jobs from Lens visualizations
+See something interesting on a visualization on your dashboard?
+Turn any *Lens* panel into a new anomaly detection job with
+the *Create anomaly detection job* menu option in your dashboards.
+
+[role="screenshot"]
+image::images/highlights-lens-anomaly.png[Create anomaly detection job option and transition to create job wizard]
+
+
+[float]
+==== Accuracy mode in Lens
+When creating *Lens* visualizations with the *Top values* function,
+you can now enable accuracy mode. This mode improves the accuracy of results for
+high-cardinality data by using more resources in your {es} cluster.
+For information, check {kibana-ref}/lens.html[Lens].
+
+[role="screenshot"]
+image::images/highlights-lens-accuracy.png["Accuracy mode in Lens", width=50%]
+
+
+
+[float]
+==== Synchronized tooltips in Lens
+
+To make it easier to compare multiple time series visualizations,
+dashboards now include a *Sync tooltips across panels* option.
+When enabled and you hover your cursor over a chart, the tooltips on all
+other related dashboard charts automatically appear at the same data point or moment in time.
+For information, check {kibana-ref}/dashboard.html[Dashboard].
+
+[role="screenshot"]
+image::images/highlights-lens-sync.png[Sync tooltip across panels option]
+
+
+[float]
+==== Collapse break down by
```

Comment:
This phrase feels like it's missing a word.  I don't know this feature, however, so maybe that's intended?

```suggestion
==== Collapse break down
```